kong/plugins/prometheus/exporter.lua

```diff
@@ -71,12 +75,43 @@ local function init()
                                          "Total bandwidth in bytes " ..
                                          "consumed per service/route in Kong",
                                          {"service", "route", "type"})
+
+
+
+  -- For all generic dbless node
+  if kong.configuration.database == "off" then
+    declarative = require("kong.db.declarative")
+
+    metrics.config_current_hash = prometheus:gauge("config_current_hash",
+                                                        "Current config hash numeric value for this node")
+  end
+
+  local role = kong.configuration.role
+  -- Hybrid mode status
+  if role == "control_plane" then
+    cp_metrics = true
+
+    clustering = require("kong.clustering")
+
+    metrics.dataplane_last_seen = prometheus:gauge("dataplane_last_seen",
+                                              "Last time data plane contacted control plane",
+                                              {"node_id", "hostname", "ip"})
+    metrics.dataplane_config_hash = prometheus:gauge("dataplane_config_hash",
+                                              "Config hash numeric value of the data plane",
+                                              {"node_id", "hostname", "ip"})
+  end
 end
 
 local function init_worker()
   prometheus:init_worker()
 end
 
+-- Convert the MD5 hex string to its numeric representation
+-- Note the following will be represented as a float instead of int64 since luajit
+-- don't like int64. Good news is prometheus uses float instead of int64 as well
```

Comment:
Yikes, how is a user suppose to interpret such a random numeric value?
@wyndigo Any thoughts?

Reply:
This is supposed to be used by compare between metrics like `count_values(config_current_hash)`